MODULE.bazel

```diff
@@ -1,8 +1,24 @@
 module(
-    name = "buildbuddy_toolchain",
-    version = "0.0.0",
+    name = "toolchains_buildbuddy",
+    version = "0.0.1",
```

Comment:
If we plan to use Publish to BCR, then we don't need to bump this.

Reply:
I want to publish the first version manually, and then use "Publish to BCR" workflows in the future. WDYT?

Reply:
That's fine. I've messed up most manual releases in the past, but the app is also not exactly forgiving. We'll figure it out :-)